frontend/public/locales/sv/common.json

```diff
@@ -17,7 +17,7 @@
   "no_results": "Inga resultat",
   "regulations": "Reglemente",
   "statutes": "Stadgar",
-  "meetingDocuments": "Möteshandlingar",
+  "meetingDocuments": "Mötesdokument",
```

Comment:
"Möteshandlingar" is historically accurate https://www.dsek.se/arkiv/moteshandlingar/
![image](https://user-images.githubusercontent.com/22683219/152601392-559d7d9d-db0d-475a-b2a4-219f04f112ce.png)
and according to Google Translate the correct translation of "Meeting documents"

Reply:
Svenska Akademien agrees with you, so I'll comply
![image](https://user-images.githubusercontent.com/97372888/152602217-7c857e70-e5f2-46cb-83f4-7d268b7732cd.png)